Makefile

```diff
@@ -1,7 +1,8 @@
 pkgs = $(shell go list ./... | grep -v /vendor/)
 files = $(shell find . -path ./vendor -prune -o -name '*.go' -print)
+ingress_binary = $(GOPATH)/bin/feed-ingress
 
-all : format vet lint test build
+all : format vet lint test build docker
```

Comment:
I'm not sure docker should be part of the default build. Doesn't it take quite a while to build?